src/Simplex_tree/benchmark/simplex_tree_stars_benchmark.cpp

```diff
@@ -0,0 +1,90 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Vincent Rouvreau
+ *
+ *    Copyright (C) 2023 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#include <gudhi/Simplex_tree.h>
+#include <gudhi/Clock.h>
+
+#include <iostream>
+#include <random>
+#include <numeric>  // for std::iota
+#include <algorithm>  // for std::shuffle
+#include <vector>
+
+std::random_device rd;
+
+template <class Vertex_handle>
+std::vector<Vertex_handle> rand_int_range(int subset_min_size,
+                                          int subset_max_size,
+                                          Vertex_handle range_max_value)
+{
+#ifdef DEBUG_TRACES
+  std::clog << "rand_int_range - subset_min_size = " << subset_min_size << " - subset_max_size = " << subset_max_size
+            << " - range_max_value = " << range_max_value << std::endl;
+#endif  // DEBUG_TRACES
+
+  std::vector<Vertex_handle> range(range_max_value);
+  std::iota(range.begin(), range.end(), 0); // range is { 0, 1, 2, ..., 99 }
+
+  std::shuffle(range.begin(), range.end(), std::mt19937 { rd() });
+
+  std::uniform_int_distribution<int> dist(subset_min_size, subset_max_size);
+  // Return a subset of shuffled range
+  // {1, 9, 13, 19, 86, 36, 83, 28, 22, 43, 12, 15}, for example
+  return std::vector<Vertex_handle> {range.begin(), range.begin() + dist(rd)};
+}
+
+
+
+template <class Stree>
+void benchmark_stars_computation() {
+  using Vertex_handle  = typename Stree::Vertex_handle;
+  using Simplex_handle = typename Stree::Simplex_handle;
+
+  std::clog << "... Simplex_tree construction" << std::endl;
+
+  Stree st;
+  const Vertex_handle VERTEX_MAX = 5000;
```

Comment:
Is the test supposed to crash if I reduce VERTEX_MAX to 500?

Reply:
I removed VERTEX_MAX and rephrase some comments on c0d24ccbb